### ISSUE TITLE
Profile edition window: setting default to None but looks selected

### ISSUE THREAD

**Issue by rassouly** [Exopy/exopy#149]
When adding some instruments, the user is required to set up some settings in the profile. When validating the connection to this instrument, the user is presented with 2 drop down menus (ObjectCombos): one for the connection and one for the settings. The issue comes from ``selected >> dial.settings``. On some machines, ``selected`` defaults to ``None`` but an item looks like it's selected for the user.
Replacing the ``>>`` operator with an ``:=`` shows that ``settings`` defaults to ``None`` (a blank option is added to the drop down).

I think that the issue comes from the following code in Enaml
```
def _default_selected(self):
        items = self.items
        if len(items) > 0:
            return items[0]
```
At the time when ``selected >> dial.settings`` is called, ``items`` has not been set yet and hence the default value of ``selected`` is None.

This issue could arise for other instances of ``ObjectCombo``.

Is there a better fix than replacing ``<<`` by ``:=``?

**Comment by MatthieuDartiailh** [comment]
The issue is not in enaml but rather in the very sloppy code I wrote. The proper way to do that is to declare the `connection` and `settings` as aliases of the `selected` member of the associated ObjectCombo. Could you add this to #150 ?

**Comment by rassouly** [comment]
Ok, I will look into this today or tomorrow.

**Comment by MatthieuDartiailh** [comment]
Thanks

**Comment by rassouly** [comment]
I tried to write a patch but I can't seem to get an alias to an attribute of a Widget inside a Conditional block to work. Any tips to get that working?
Alternatively, how does this solution differ from ``selected := dial.settings`` (which does work)?

**Comment by MatthieuDartiailh** [comment]
I missed the `Conditional` (I like those guys less and less...) A two-way synchronization is fine, so let's go for this. Also could you fix the `selected` handler for the case of multiple drivers, it looks wrong (selected_item does not exist).

**Comment by MatthieuDartiailh** [comment]
I believe it is closed by #150 so closing
